src/js/components/Form/stories/FieldWithChildren.stories.js

```diff
@@ -88,7 +88,7 @@ export const FieldWithChildren = () => (
             options={['morning', 'evening']}
           />
         </FormField>
-        <FormField label="Size" htmlFor="size" name="size">
+        <FormField label="Size" htmlFor="size__input" name="size">
```

Comment:
Just to confirm, these story changes aren't required for this functionality (because now grommet would add the __input automatically), but we're adding it since we document as a best practice. Correct?

Reply:
exactly.

Reply:
thats a good point. Since we add automatically now should we change the docs? or still show this in the stories?
@taysea @jcfilben

Reply:
yeah I wonder to keep it clean we remove in the story if we update docs

Reply:
removed the `__input` in stories